epochStart/metachain/stakingDataProvider.go

```diff
@@ -377,7 +458,7 @@ func (sdp *stakingDataProvider) createMapBLSKeyStatus(validatorsInfo state.Shard
 	return mapBLSKeyStatus, nil
 }
 
-func (sdp *stakingDataProvider) selectKeysToUnStake(sortedKeys map[string][][]byte, numToSelect int64) [][]byte {
+func (sdp *stakingDataProvider) selectKeysToUnStake(sortedKeys map[string][][]byte, numToSelect int64) ([][]byte, int) {
```

Comment:
maybe this function can be renamed

Reply:
IMHO, the name is good and reflects its purpose; I would leave it as it is, unless you have a stronger opinion on why it should be renamed

`selectedKeys, removedValidators := sdp.selectKeysToUnStake(sortedKeys, numKeysToUnStake)`